Tribler/Core/Modules/MetadataStore/OrmBindings/channel_node.py

```diff
@@ -19,7 +19,8 @@
 JUST_RECEIVED = 3
 UPDATE_AVAILABLE = 4
 PREVIEW_UPDATE_AVAILABLE = 5
-LEGACY_ENTRY = 6
+UPDATED = 6
```

Comment:
Could you please add some comments here to indicate what each status exactly means?

Reply:
Good point, I'll do that.